src/config/ws.ts

```diff
@@ -0,0 +1,52 @@
+import WebSocket from "ws";
+
+export interface WebSocketMessage {
+  event: string;
+  payload: string;
+}
+
+import { getList } from "../repositories/user.repository";
+import { IncomingMessage } from "http";
+
+export const connectedUsers: Map<string, Set<WebSocket>> = new Map();
+const connections: Map<WebSocket, string> = new Map();
+
+export const connection = async (ws: WebSocket, req: IncomingMessage) => {
+  if (!req.headers.userid) {
```

Comment:
why do we add it at header? Maybe better use 
ws.on("message"  and put it in that section. userid can put inside message